.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,19 @@
+name: 'Continuous Integration'
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  check-dist:
+    name: Check dist/ directory
+    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@a8533f184b279cfc1b2dd6a96ed2f097ccf81189
+    with:
+      node-version: "20.x"
+  test:
+    name: Test
+    uses: actions/reusable-workflows/.github/workflows/basic-validation.yml@a8533f184b279cfc1b2dd6a96ed2f097ccf81189
+    with:
+      node-version: "20.x"
```

```suggestion
  check-dist:
    name: Check dist/ directory
    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@967035ce963867fb956a309c9b67512314bc7c1f
    with:
      node-version: "20.x"

  test:
    name: Test
    uses: actions/reusable-workflows/.github/workflows/basic-validation.yml@967035ce963867fb956a309c9b67512314bc7c1f
    with:
      node-version: "20.x"
```

Use a more recent and compatible SHA.